tests/systemtests/Makefile

```diff
@@ -5,7 +5,7 @@ WAIT_TIME ?= 45s
 all: test
 
 test:
-	go test -mod=readonly -failfast -tags='system_test' ./... --wait-time=$(WAIT_TIME) --verbose $(if $(findstring v2,$(COSMOS_BUILD_OPTIONS)),--binary=simdv2)
+	go test -mod=readonly -failfast -tags='system_test $(if $(findstring v2,$(COSMOS_BUILD_OPTIONS)),v2)' ./... --wait-time=$(WAIT_TIME) --verbose $(if $(findstring v2,$(COSMOS_BUILD_OPTIONS)),--binary=simdv2)
```

Comment:
In https://github.com/cosmos/cosmos-sdk/pull/21539/files#diff-d430b065976a77ab03608cab74b5c1db97ec128880faab2ea96b24db767ed75eR16-R20, we added a `isV2()` function, maybe let's use that instead of build tags and just use `t.Skip` within the test.